bin/release.sh

```diff
@@ -36,24 +50,21 @@ npx lerna run build \
   --ignore @bugsnag/plugin-electron-app \
   --ignore @bugsnag/plugin-electron-client-state-persistence
 
-# check if CDN packages changed – if they didn't we don't need to upload to the CDN
-BROWSER_PACKAGE_CHANGED=$(npx lerna changed --parseable | grep -c packages/js$ || test $? = 1;)
-WORKER_PACKAGE_CHANGED=$(npx lerna changed --parseable | grep -c packages/web-worker$ || test $? = 1;)
+# push latest git tag
+TAG=$(git describe --tags --abbrev=0)
+git push origin $TAG
```

Comment:
```suggestion
git push origin "$TAG"
```

Reply:
Do we need to push 1 tag specifically, or would `git push --tags` be OK?

Reply:
I think push --tags would be okay actually, as it's from a fresh docker container